src/twisted/logger/test/test_format.py

```diff
@@ -429,3 +430,94 @@ class Unformattable(object):
 
     def __repr__(self):
         return str(1 / 0)
+
+
+
+class CapturedError(Exception):
+    """
+    A captured error for use in format tests.
+    """
+
+
+
+class FormatEventWithTracebackTests(unittest.TestCase):
+    """
+    Tests for L{formatEventWithTraceback}, all of which ensure that the
+    returned type is a UTF-8.
```

Comment:
This is an incomplete sentence.  I think it should be, "... all of which ensure that the returned type is UTF-8 decoded text."